packages/now-static-build/src/frameworks.ts

```diff
@@ -263,6 +263,53 @@ const frameworkList: Framework[] = [
       },
     ],
   },
+  {
+    name: 'Ionic React',
+    slug: 'ionic-react',
+    dependency: '@ionic/react',
+    buildCommand: 'react-scripts build',
+    getOutputDirName: async () => 'build',
```

Comment:
Is it `build` or `public` because `@now/frameworks` has `public`.

cc @AndyBitz

Reply:
It's `build` for `ionic-react`